plugins/lime-plugin-fbw/src/FbwPage.spec.js

```diff
@@ -175,8 +176,56 @@ describe('Fbw Join Network Page', () => {
             expect(await screen.findByText('Error downloading lime assets')).toBeInTheDocument();
         });
     });
+
+    it('shows loader when status is scanning', async () => {
+        await advanceToJoinNetwork()
+        await waitForExpect(async () => {
+            expect(await screen.findByTestId('loading')).toBeInTheDocument();
+        });
+    })
+
+    it('not shows loader when status is scanning', async () => {
+        scanStatus.mockImplementation(async () => {
+            allScanCases.status = 'scanned'
+            return allScanCases
+        })
+
+        await advanceToJoinNetwork()
+        expect(await screen.findByTestId('loading')).not.toBeInTheDocument();
```

Comment:
findBy should not be used when your are expecting an element not to be present.
Instead follow: https://testing-library.com/docs/guide-disappearance/

Reply:
Using `queryByTestId` works fine also 4ae00b6